### PR TITLE
Take also environment variables

### PR DESCRIPTION
**Pull request by mdemri**
Changed Commander usage for CLI with Yargs.
Which takes also Environment Variables.

Purpose: Make Enigme-p2p work as a standalone server, with environment variables injection

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (lacabra) [inline .vscode/launch.json]: I would argue that this file does not belong to the repo. This is a configuration file of your particular setup with Visual Studio Code. I'd rather put `.vscode/*` in `.gitignore` and remove this file from this PR.

Comment (mdemri) [reply .vscode/launch.json]: OK

File: .gitignore [hunk 2]
```diff
@@ -253,3 +249,6 @@ test/*.deletedb
 *.deletedb
 src/cli/*.deletedb
 
+# pyenv python version (for node-gyp builds) (pyenv local 2.7.15)
```
Comment (lenak25) [inline .gitignore]: Could you please elaborate on this?

File: package.json [hunk 3]
```diff
@@ -10,6 +10,8 @@
     "temp-dir": "/tmp/enigma-p2p"
   },
   "scripts": {
+    "start": "node src/server.js",
```
Comment (lenak25) [inline package.json]: Nice addition!
Does it work with no args?

File: src/cli/cli_app.js [hunk 4]
```diff
@@ -441,13 +512,13 @@ class CLI {
...
-        console.log('XXX no such command XXX ');
+        log.info('XXX no such command XXX ');
       }
     });
     return this;
   }
 }
 // TODO:: the CLI starts automatically for now
-const cli = new CLI().start();
+new CLI().start();
```
Comment (lenak25) [inline src/cli/cli_app.js]: This should be removed.. It happens twice now

File: package.json (hunk 3 above)
Comment (mdemri) [reply package.json]: yes. only env vars

File: .gitignore (hunk 2 above)
Comment (mdemri) [reply .gitignore]: node-gyp complition requires python. i use a tool called pyenv, like venv but better.
to use different python version in each project.
Not relevant, only .gitignore.

File: src/cli/cli_app.js (hunk 4 above)
Comment (mdemri) [reply src/cli/cli_app.js]: you right.

Comment (mdemri) [reply src/cli/cli_app.js]: made change in server.js

**Comment by assafmo** [comment]
Itzik said this PR isn't needed anymore.

**Pull request closed without merging**